src/_includes/core-nodes/http-request-use-case.md

```diff
@@ -0,0 +1,85 @@
+## What are HTTP request nodes used for in Node-RED
+
+In Node-RED, an HTTP Request node is a node that allows you to make HTTP requests to external servers or services. This node allows you to interact with web services, APIs, or any other HTTP-based endpoints.
+
+When you configure an HTTP Request node, you typically specify the method (GET, POST, PUT, DELETE, etc.), the URL of the endpoint you want to communicate with, any headers you need to include, and the payload if applicable. Once configured, this node will send the HTTP request when triggered by an incoming message or event.
+
+## Configuring HTTP Request node
+
+Below, you'll find a range of settings to tailor HTTP requests to fit the needs of different APIs or web services. Depending on the service you're working with, some options might be crucial, while others could be optional.
+
+- **Method:** Select the HTTP method for the request (e.g., GET, POST, PUT, DELETE). You can dynamically set it using `msg.method`.
+- **URL:** Specify the endpoint URL to communicate with. Dynamic URL setting is allowed using `msg.url`. additionally, if you want to construct a URL with the message's property you can utilize Mustache-style tags. ex - `example.com/{{{topic}}}`, it will have the value of `msg.topic` automatically inserted. Using {{{...}}} prevents mustache from escaping characters like / & etc.
```

Comment:
Hold on, I thought you want escaping and thus use `{{....}}`?

Reply:
Both are working, but I've used `{{{}}}` because it's in the current documentation for the HTTP request core node. Which one should we use?

Reply:
Done !